common/lib/capa/capa/templates/optioninput.html

```diff
@@ -30,6 +30,6 @@
   </div>
   <p class="answer" id="answer_${id}"></p>
   % if msg:
-      <span class="message">${HTML(msg)}</span>
+      <span class="message" aria-describedby="label_${id}" aria-live="polite" tabindex="-1">${HTML(msg)}</span>
```

Comment:
And if there are particular templates where there is no label (or it is unclear what the label is), please add a comment in the PR about them and we can discuss them individually.